netci.groovy

```diff
@@ -34,6 +34,10 @@ platformList.each { platform ->
     else if (os == 'Ubuntu') {
         buildCommand = "./build.sh --skip-prereqs --configuration ${configuration} --docker ubuntu.14.04 --targets Default"
     }
+	else if (os == 'Linux') {
+		os = 'Ubuntu16.04';
```

Comment:
The reason that core-setup uses RHEL 7.2 is that it needs to build native code and we want to do that on a machine that has the oldest version of GLIBC we support (and in our cases that's on RHEL 7).  In CLI's case, you aren't building any native code, so you should be able to do your managed linux build anywhere.